apps/asap-server/test/handlers/teams/index-handler.test.ts

```diff
@@ -0,0 +1,50 @@
+import { EventBridgeEvent } from 'aws-lambda';
+import {
+  indexHandlerFactory,
+  SquidexWebhookTeamPayload,
+} from '../../../src/handlers/teams/index-handler';
+import { TeamsEventType } from '../../../src/handlers/webhooks/webhook-teams';
+import { createEventBridgeEventMock } from '../../helpers/events';
+import {
+  algoliaClientMock,
+  algoliaIndexMock,
+} from '../../mocks/algolia-client.mock';
+import { teamControllerMock } from '../../mocks/team-controller.mock';
+import { teamResponse } from '../../fixtures/teams.fixtures';
+
+describe('Team Research Outputs Index', () => {
+  const indexHandler = indexHandlerFactory(
+    teamControllerMock,
+    algoliaClientMock,
+  );
+
+  afterEach(() => jest.clearAllMocks());
+
+  test('Should fetch the team and create a record in Algolia for every research output', async () => {
```

Comment:
maybe add a test that confirms that algolia client is not called when the team has no associated researchOutputs?